test/lib.ts

```diff
@@ -54,7 +54,7 @@ export function setMultiplePeersAround(
   position: Position3D,
   qty: number,
   idGenerator: IdGenerator = sequentialIdGenerator("P"),
-  offset: Position3D = [20, 0, 20]
+  offset: Position3D = [10, 0, 10]
```

Comment:
Curious, why was this change necessary?

Reply:
The test was a little flaky, because it has random positions. Reducing the offset seems to make it more reliable.

Eventually we may want to use a known seed to make the tests deterministic.